.gitignore

```diff
@@ -0,0 +1,4 @@
+tests/expected.json
```

Comment:
if these are for bioentities, could we preprend that somewhere? otherwise it will be confusing when we add the analytics part.

Reply:
prefixes used in this commit. https://github.com/ebi-gene-expression-group/index-gxa/pull/2/commits/69314e4fae286abad09be4ca44ba4762152f7570